pages/domains/domains_before.md

```diff
@@ -50,54 +54,62 @@ You’ll choose from the list below.
 
-### Authorizing official
+### Your authorizing official
 
-Your authorizing official is a person within your organization who can authorize your domain request. This person must be in a role of significant, executive responsibility within the organization. [Read more about who can serve as an authorizing official](../eligibility/#you-must-have-approval-from-an-authorizing-official-within-your-organization).
+Your authorizing official is a person within your organization who can authorize your domain request. This person must be in a role of significant, executive responsibility within the organization. Read more about [who can serve as an authorizing official](../eligibility/#you-must-have-approval-from-an-authorizing-official-within-your-organization).
 
 What we’ll need to know about your authorizing official:
 - Name
 - Role in your organization
 - Email address
-- Phone number
 
-After you submit your request, we’ll contact your authorizing official to double check that they approve it.
+We typically don’t reach out to the authorizing official, but if contact is necessary, our practice is to coordinate with you, the requestor, first. 
 
-### Purpose of the .gov domain
+### Current websites for your organization
 
-We’ll ask you to explain how you plan to use your .gov domain. Will you use it for a website and/or email? Are you moving your website from another top-level domain (like .com or .org)? Read about [activities that are prohibited on .gov domains](../requirements/#what-you-can%E2%80%99t-do-with-.gov-domains).
-
-### Current website for your organization
-
-We’ll ask about your organization’s current public website. We can better evaluate your domain request if we know about domains you’re using. If you already have any .gov domains please include them.
+We’ll ask about your organization’s current public websites. We can better evaluate your domain request if we know about domains you’re already using. If you already have a .gov domain, include that in your list.
 
 ### .Gov domain you want
 
-Here’s the part where you’ll tell us the .gov domain you want. We’ll try to give you your preferred domain. We first need to make sure your request meets our requirements. We’ll work with you to find the best domain for your organization.
+Here’s the part where you’ll tell us the .gov domain you want. We’ll try to give you your preferred domain, but we first need to make sure it meets our requirements. We’ll work with you to find the best domain for your organization.
 
 {% include 'content-blocks/general_domain_requirements.md' %}
 
-[Read more about our domain name requirements](../requirements/).
+Read more about [our domain name requirements](../requirements/).
+
+### Purpose of your domain
+
+We’ll ask you to explain how you plan to use your .gov domain. Will you use it for a website and/or email? Read about [activities that are prohibited on .gov domains](../requirements/#what-you-can%E2%80%99t-do-with-.gov-domains).
```

Comment:
For the "activities that are prohibited on .gov domains" link, need to use **#what-you-can%E2%80%99t-do-with-a-.gov-domain** instead of **#what-you-can%E2%80%99t-do-with-.gov-domains** (missing the "a" in "a-.gov")

Reply:
Note that `%E2%80%99` is an issue with prime mark vs. single quote. If we fix it on the div id and this anchor we won't have an ugly URL.